circuit_knitting/cutting/cutting_decomposition.py

```diff
@@ -13,9 +13,9 @@
 
 from __future__ import annotations
 
+from typing import NamedTuple
 from collections import defaultdict
 from collections.abc import Sequence, Hashable
-from typing import NamedTuple
```

Comment:
It'd be nice to revert this change, which would mean one fewer file touched.  But not a big deal.  I am splitting hairs.